winappdbg/debug.py

```diff
@@ -1079,16 +1082,21 @@ def next(self):
             event handler raises an exception nobody catches.
         """
         try:
-            event = self.wait()  # NOQA
+            try:
+                event = self.wait(time_limit_in_seconds * 1000)  # NOQA
+            except WindowsError:
+                if time_limit_in_seconds == 0:
+                    pass

```

Comment:
Wouldn't this make it ignore all exceptions instead of raising them again?

Reply:
Yes, so it seems, noob mistake, sorry

Reply:
No prob! That's what peer review is for! :)